.prospector.yml

```diff
@@ -13,6 +13,7 @@ ignore-paths:
   - docs
   - niceday_client/niceday_client/__init__.py
   - db/dbschema/migrations
+
```

Comment:
This empty line is definitely critical for the functionality added to this PR

Reply:
It brings harmony to the config file :)